plugins/package-managers/node/src/main/kotlin/Yarn2.kt

```diff
@@ -100,6 +103,11 @@ class Yarn2(
          */
         const val OPTION_DISABLE_REGISTRY_CERTIFICATE_VERIFICATION = "disableRegistryCertificateVerification"
 
+        /**
+         * The name of the option that determines whether Yarn 2+ was installed via Corepack.
+         */
+        const val OPTION_COREPACK_ENABLED = "corepackEnabled"
```

Comment:
commit-message: Can you explain _why_ you choose to implement this as an option as opposed to adding detection whether Corepack is enabled? Looking at [1] it says:
```
 You can quickly check whether Corepack is enabled by
running `yarn exec env`: if you get a path as output, Corepack
 is properly installed. If not, something may be messing with
how the shims are installed.
```

^[1]: https://yarnpkg.com/corepack

Reply:
See my [other comment](https://github.com/oss-review-toolkit/ort/pull/8703#discussion_r1616777462).